en/python_installation/instructions.md

```diff
@@ -16,15 +16,15 @@ First check whether your computer is running a 32-bit version or a 64-bit versio
 
 You can download Python for Windows from the website https://www.python.org/downloads/windows/. Click on the "Latest Python 3 Release - Python x.x.x" link. If your computer is running a **64-bit** version of Windows, download the **Windows x86-64 executable installer**. Otherwise, download the **Windows x86 executable installer**. After downloading the installer, you should run it (double-click on it) and follow the instructions there.
 
-One thing to watch out for: During the installation, you will notice a window marked "Setup". Make sure you tick the "Add Python 3.6 to PATH" or 'Add Python to your environment variables" checkbox and click on "Install Now", as shown here (it may look a bit different if you are installing a different version):
+One thing to watch out for: During the installation, you will notice a window marked "Setup". Make sure you tick the "Add Python {{ book.py_version }} to PATH" or 'Add Python to your environment variables" checkbox and click on "Install Now", as shown here (it may look a bit different if you are installing a different version):
 
 ![Don't forget to add Python to the Path](../python_installation/images/python-installation-options.png)
 
 When the installation completes, you may see a dialog box with a link you can follow to learn more about Python or about the version you installed. Close or cancel that dialog -- you'll be learning more in this tutorial!
 
-Note: if you are using an older version of Windows (7, Vista, or any older version) and the Python 3.6.x installer fails with an error, you can try either:
+Note: if you are using an older version of Windows (7, Vista, or any older version) and the Python {{ book.py_version }}.x installer fails with an error, you can try either:
 1. install all Windows Updates and try to install Python again; or
-2. install an [older version of Python](https://www.python.org/downloads/windows/), e.g., [3.4.6](https://www.python.org/downloads/release/python-346/).
+2. install an [older version of Python](https://www.python.org/downloads/windows/), e.g., [{{ book.py_min_version }}](https://www.python.org/downloads/release/python-346/).
```

Comment:
The second URL is for Python 3.4.6 and should probably be changed to conform with `book.py_min_version`.

Reply:
Solution 1: Remove URL and keep only version text

Solution 2: Give static URL updated with each minimum version changes and set it to 3.6.0 for now and set URL

Solution 3. Set all minimum version as static text (3.6.0) as well as URL

@das-g  What do you think we can go forward with?

Reply:
@das-g Found a mistake. Django 3 only supports 3.6. So we can not provide backward compatibility with Python 3.4. So why do we need this section if we are using the 3.8.6 version which is providing everything it can as the primary version we are using?

```
Note: if you are using an older version of Windows (7, Vista, or any older version) and the Python {{ book.py_version }} installer fails with an error, you can try either:
1. install all Windows Updates and try to install Python again; or
2. install an [older version of Python](https://www.python.org/downloads/windows/), e.g., [3.4.10](https://www.python.org/downloads/release/python-3410/).

If you install an older version of Python, the installation screen may look a bit different than shown above. Make sure you scroll down to see "Add python.exe to Path", then click the button on the left and pick "Will be installed on local hard drive":

![Add Python to the Path, older versions](../python_installation/images/add_python_to_windows_path.png)
```

So the above part is no more needed at all. Maybe we can try the following:

```
Note: if you are using an older version of Windows (7, Vista, or any older version) and the Python {{ book.py_version }} installer fails with an error, then install all Windows Updates and try to install Python again
```

Reply:
Yeah, let's modify it that way, i.e., remove option 2. See also https://github.com/DjangoGirls/tutorial/pull/1683#discussion_r523408737

Reply:
@das-g Here is an updated version of same.

```
Note: If you are using an older version of Windows (7, Vista, or any older version) and the Python {{ book.py_version }} installer fails with an error, then install all Windows Updates and try to install Python again. If you still have the error, try installing Python version {{ book.py_min_version }} from [Python.org](https://www.python.org/downloads/windows/).

> Django {{ book.django_version }} need {{ book.py_min_version }} or greater which does not support Windows XP or earlier versions. If you are using Windows XP or earlier, please upgrade to your Windows OS.
```

Reply:
I can see

>     > Django {{ book.django_version }} need {{ book.py_min_version }} or greater which does not support Windows XP or earlier versions. If you are using Windows XP or earlier, please upgrade to your Windows OS.

on https://django.nikz.in/en/python_installation/#python-installation, but not in the diff of the PR here. Did you maybe make that addition locally, without pushing it to the branch on your GitHub repo?

Do we need that note at all? If so, I think it should be

>     > Django {{ book.django_version }} needs Python {{ book.py_min_version }} or greater, which does not support Windows XP or earlier versions.

Changes:
- "need**s**" instead of "need"
- "**Python** {{ book.py_min_version }}" instead of just "{{ book.py_min_version }}"
- leave out the second sentence. How to resolve the issue is IMO up to the participant or whoever maintains the participant's computer. Some might want to buy a newer Windows, others might decide to switch to Linux.

Reply:
Yes i didn't pushed it to PR since I am not sure. I changed it in local master only for deploy. I'll push it with your changes mentioned to this PR. Looks good.

Reply:
Resolved with commits https://github.com/DjangoGirls/tutorial/pull/1683/commits/0aab1eb0df8e2b31486d4dd2f21adab47be5d3d7 and https://github.com/DjangoGirls/tutorial/pull/1683/commits/f99a47c1f1960df4cf6361627ac9a12f3bee7171